frontend/src/scenes/events/EventsVolumeTable.tsx

```diff
@@ -38,6 +40,8 @@ export function VolumeTable({
 }): JSX.Element {
     const [searchTerm, setSearchTerm] = useState(false as string | false)
     const [dataWithWarnings, setDataWithWarnings] = useState([] as VolumeTableRecord[])
+    const { user } = useValues(userLogic)
+    const isPremiumUser = user?.organization?.available_features?.includes('dashboard_collaboration')
```

Comment:
Nit: confusing, maybe `hasTaxonomyFeatures` ? Also I think `dashboard_collaboration` is not the right permission to check for